Testing/PythonTests/geodesic.py

```diff
@@ -0,0 +1,15 @@
+import os
+import sys
+from shapeworks import *
+
+def geodesicTest():
+  mesh = Mesh(os.environ["DATA"] + "/ellipsoid_0.ply")
+  dist = mesh.geodesicDistance(10, 20)
+
+  return abs(dist - 1.0083) < 1e-4
+
+val = geodesicTest()
```

Comment:
Please use try/except for this and all other python tests.

Example:
```python
try:
  if not testFunction():
    print("testName failed")
    sys.exit(1)
except RuntimeError:
  print("testName failed (exception)")
  sys.exit(1)
```

Reply:
Done